docs/SmartStore.md

```diff
@@ -1,6 +1,6 @@
 # SmartStore Resource Guide
 
-*NOTE: The below method is a temporary way of installing SmartStore configuration & indexes. Starting from the Splunk Operator release 1.0.2, an enhanced App installation framework is introduced which is the recommended method to install SmartStore indexes & configuration.*
+*NOTE: The below method is a temporary way of installing SmartStore configuration & indexes. Starting from the Splunk Operator release 1.0.2, an enhanced App installation framework is introduced which is the recommended method to install SmartStore indexes & configuration. The below method can still be used if the customer wants to avoid storing the S3 secret and access keys in the S3 buckets(via the App installation framework)*
```

Comment:
How about modifying something like:
`The following method may still be used to specify the S3 access keys, which avoids storing them in the S3 buckets (via the app installation framework).`